src/main/java/org/jenkinsci/plugins/github/pullrequest/GitHubPRTrigger.java

```diff
@@ -422,7 +424,8 @@ public DescriptorImpl() {
 
         @Override
         public boolean isApplicable(Item item) {
-            return item instanceof AbstractProject;
+            return item instanceof Job && SCMTriggerItem.SCMTriggerItems.asSCMTriggerItem(item) != null
+                    && item instanceof ParameterizedJobMixIn.ParameterizedJob;
```

Comment:
afair copy-pasted from GH plugin, but work of triggering is based on parameter actions, so probably it right.